barretenberg/cpp/src/barretenberg/benchmark/basics_bench/basics.bench.cpp

```diff
@@ -466,6 +467,19 @@ void pippenger(State& state)
         benchmark::DoNotOptimize(ck->commit(pol));
     }
 }
+
+void bn254fr_random(State& state)
+{
+    numeric::RNG& engine = numeric::get_randomness();
+    for (auto _ : state) {
+        state.PauseTiming();
+        size_t num_cycles = 1 << static_cast<size_t>(state.range(0));
```

Comment:
minor nit:  numeric literals like `1` are usually interpreted as `int` types. Might make the code ever so slightly more readable by making this `1UL` (32 bit unsigned long). Up to you though, this isa very petty comment.

Reply:
Thanks